### ISSUE TITLE
Add APCA settings

### ISSUE THREAD

**Issue by TheJaredWilcurt** [vue-dev-tools-accessibility/v0#16]
https://www.npmjs.com/package/apca-check

Looks like we should be able to dynamically toggle between:

* WCAG 2 AA
* APCA Bronze
* APCA Silver

**Comment by TheJaredWilcurt** [comment]
Check Bronze/Silver against these colors

```
LIGHT MODE PILL
Element has insufficient color contrast of 3.65:1
(text color: #FFFFFF, background color: #16986D, font size: 9.0pt (12px), font weight: normal).
Expected contrast ratio of 4.5:1.

    Suggested text color: #1F1F1F
    Suggested background color: #138660
```
```
DARK MODE PILL
Element has insufficient color contrast of 2.39:1
(text color: #FFFFFF, background color: #36BC90, font size: 9.0pt (12px), font weight: normal).
Expected contrast ratio of 4.5:1.

    Suggested text color: #3D3D3D
    Suggested background color: #268364
```

**Comment by TheJaredWilcurt** [comment]
Results for Bronze:
```
LIGHT MODE PILL - APCA Bronze
Element has insufficient APCA bronze level contrast of 69.12Lc
(foreground color: #FFFFFF, background color: #16986D, font size: 9.0pt (12px), font weight: 400).
Expected minimum APCA lightness contrast of 75Lc.
```
```
DARK MODE PILL - APCA Bronze
Element has insufficient APCA bronze level contrast of 51.82Lc
(foreground color: #FFFFFF, background color: #36BC90, font size: 9.0pt (12px), font weight: 400).
Expected minimum APCA lightness contrast of 75Lc.
```
```
LIGHT MODE PILL - APCA Silver
Element has insufficient APCA silver level contrast of 69.12Lc
(foreground color: #FFFFFF, background color: #16986D, font size: 9.0pt (12px), font weight: 400).
Increase font size and/or font weight to meet APCA conformance minimums.
```
```
DARK MODE PILL - APCA Silver
Element has insufficient APCA silver level contrast of 51.82Lc
(foreground color: #FFFFFF, background color: #36BC90, font size: 9.0pt (12px), font weight: 400).
Increase font size and/or font weight to meet APCA conformance minimums. 
```

APCA Silver Conformance level has no Lc because it requires a minimum font-size of 14px, according to the Lookup Table (LUT).

* https://github.com/StackExchange/apca-check/blob/main/src/apca-silver-plus.ts
